packages/mds-agency/request-handlers.ts

```diff
@@ -314,15 +341,14 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
   }
 }
 
-export const submitVehicleTelemetry = async (req: AgencyApiRequest, res: AgencyApiResponse) => {
+export const submitVehicleTelemetry = async (req: AgencyApiRequest, res: AgencySubmitVehicleTelemetryResponse) => {
   const start = Date.now()
 
   const { data } = req.body
   const { provider_id } = res.locals
   if (!provider_id) {
     res.status(400).send({
-      error: 'bad_param',
-      error_description: 'bad or missing provider_id'
+      error: new BadParamsError('bad or missing provider_id')
```

Comment:
Should have a `error_description` field as well, considering we had it before

Reply:
Also `error` needs to be a string of something like `bad_param`, see the [Agency Telemetry](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/agency#vehicles---telemetry) spec

Reply:
Can we just merge this versioning one and I'll cut another PR to address all the places the agency code diverges from the spec? Some of the `200` responses also diverge from it.